modules/swagger-codegen/src/main/java/io/swagger/codegen/languages/FlaskConnexionCodegen.java

```diff
@@ -654,7 +655,23 @@ public String toModelImport(String name) {
             if (!"".equals(modelPackage())) {
                 modelImport += modelPackage() + ".";
             }
-            modelImport += toModelFilename(name)+ " import " + name;
+
+            // with modelNamePrefix/modelNameSuffix, name already includes
```

Comment:
This is a bit ugly from the code design side of things, but since it works, l don't mind that much.

Reply:
Agreed. I suspect other language generators suffer from the same bug (#7866), and it seems the general assumption is that the name parameter is unprocessed. Perhaps a better fix would be to make sure it actually is. However, doing so would require careful review of the 50+ other language generator files. Thanks @frol.